dex-operator/README.md

```diff
@@ -0,0 +1,48 @@
+# Dex Operator
+
+## Overview
+This Dex operator is an operator built with the [Operator SDK][operator_sdk]. Currently the operator supports scaling the numbers of pods on the mock connector and future work involves allowing the connectors to be configured using the operator CRD.
```

Comment:
You are essentially scaling the Dex pods not the connector. This statement is a little misleading. Please clarify that the Dex pod is essentially the server.